### PR TITLE
footer content

### LINKED ISSUE #11: Add footer content

### PR DESCRIPTION
**Pull request by Olayinkaboluwatife**
fixed #11

### CODE REVIEW

File: html/app.html [hunk 1]
```diff
@@ -22,7 +22,65 @@
...
+          </ul>
+        </section>
+
+        <section>
+          <h4>Subscribe To Our Newsletter</h4>
+          <p>More recently with desktop publishing like aldus pagemaker including</p>
+        </section>
+
+        <section>
+          <form>
```
Comment (yunushabeeb) [inline html/app.html]: No action or method in form.

File: html/app.html [hunk 2]
```diff
@@ -22,7 +22,65 @@
...
+          </form>
+        </section>
+
+        <section>
+          <p>@2021 Crimson, Inc All rights reserved </p>
+          <img src="../img/footer/facebook.png">
+          <img src="../img/footer/github.png">
+          <img src="../img/footer/linkedin.png">
+          <img src="../img/footer/messager.png">
+          <img src="../img/footer/techlogo.png">
```
Comment (yunushabeeb) [inline html/app.html]: All image element not having an alternative message.

File: html/app.html [hunk 3]
```diff
@@ -22,7 +22,65 @@
...
+          <ul>
+            <li><a href="">Pricing</a></li>
+            <li><a href="">Documentation</a></li>
+            <li><a href="">Sitemap</a></li>
+            <li><a href="">Documentation</a></li>
+            <li><a href="">Documentation</a></li>
+          </ul>
+        </section>
+
+        <section id="Company">
```
Comment (yunushabeeb) [inline html/app.html]: Multiple sections can not have same id.

**Review by yunushabeeb — COMMENTED** [review]
Check through the html file. Some comments are waiting for you there.

File: html/app.html [hunk 4]
```diff
@@ -22,7 +22,65 @@
...
+          <ul>
+            <li><a href="">Pricing</a></li>
+            <li><a href="">Documentation</a></li>
+            <li><a href="">Sitemap</a></li>
+            <li><a href="">Documentation</a></li>
+            <li><a href="">Documentation</a></li>
+          </ul>
+        </section>
+
+        <section id="new">
```
Comment (d-code-h) [inline html/app.html:43]: Make the id semantically sound. New what?

Comment (yunushabeeb) [inline html/app.html]: Make the id semantically sound. New what?

File: html/app.html [hunk 5]
```diff
@@ -22,7 +22,65 @@
...
+          </ul>
+        </section>
+
+        <section id="Company">
+          <h5>Company</h5>
+          <ul>
+            <li><a href="">About Us</a></li>
+            <li><a href="">Partners</a></li>
+            <li><a href="">Testimonials</a></li>
+            <li><a href="">Portfolio</a></li>
```
Comment (yunushabeeb) [inline html/app.html]: Link that has no destination should have a # href to avoid broken pages.

File: html/app.html [hunk 6]
```diff
@@ -22,7 +22,65 @@
...
+          </ul>
+        </section>
+
+        <section id="new">
+          <h5>Company</h5>
+          <ul>
+            <li><a href="">About Us</a></li>
+            <li><a href="">Partners</a></li>
+            <li><a href="">Testimonials</a></li>
+            <li><a href="">Portfolio</a></li>
```
Comment (yunushabeeb) [inline html/app.html]: No references.

File: html/app.html [hunk 7]
```diff
@@ -22,7 +22,65 @@
...
+        </section>
+
+        <section class="support">
+          <h5>Support</h5>
+          <ul>
+            <li><a href="">Pricing</a></li>
+            <li><a href="">Documentation</a></li>
+            <li><a href="">Sitemap</a></li>
+            <li><a href="">Documentation</a></li>
+            <li><a href="">Documentation</a></li>
```
Comment (yunushabeeb) [inline html/app.html]: No references.

**Pull request merged**